src/power_grid_model_io/utils/uuid_excel_cvtr.py

```diff
@@ -159,6 +159,7 @@ def insert_or_update_number_column(
         number (str): "Number" or "Nummer" depending on the language
     """
     new_column_name = guid_column.replace("GUID", number)
+    # pylint: disable=duplicate-code
```

Comment:
cfr. my statement about the dictionary: both the dictionary itself and the sheet name language extraction may be moved to `utils` (which makes perfect sense). that would remove this `pylint` warning